backend/api/go_http_client/experiment_client/experiment_service/experiment_service_client.go

```diff
@@ -39,7 +39,7 @@ type Client struct {
 }
 
 /*
-ArchiveExperiment archives an experiment runs and jobs inside the experiment will be archived as well
+ArchiveExperiment archives an experiment and the experiment s runs and jobs
```

Comment:
experiment s -> experiment's

Please update lines 42, 100, and 187 for the same issue.

Reply:
The text here in go_http_client directory is auto-generated by the swagger tools. The auto-generation is based on the .proto files, and this file is in generally not edited manually, since any manual revision to this file will be overridden when the auto-generation process is taking place again.

Therefore, we usually only make sure that the text in .proto files is good for reading. And ignore the text in go_http_client folder.